rules/windows/process_creation/proc_creation_win_cmd_dosfuscation.yml

```diff
@@ -21,6 +22,7 @@ detection:
             - ' s^et '
             - ' s^e^t '
             - ' se^t '
+            - ':~'
```

Comment:
This is too dangerous. Legitimate scripts use that quite often. 
We could cover expressions like that in a separate rule with lower level.